operators/src/source/ogr_source.rs

```diff
@@ -712,35 +708,41 @@ where
 
             match data_type {
                 FeatureDataType::Text => {
+                    #[allow(clippy::match_same_arms)]
                     let text_option = match field {
-                        Ok(FieldValue::IntegerValue(v)) => Some(v.to_string()),
-                        Ok(FieldValue::Integer64Value(v)) => Some(v.to_string()),
-                        Ok(FieldValue::StringValue(s)) => Some(s),
-                        Ok(FieldValue::RealValue(v)) => Some(v.to_string()),
-                        Ok(_) => todo!("handle other types"),
+                        Ok(Some(FieldValue::IntegerValue(v))) => Some(v.to_string()),
+                        Ok(Some(FieldValue::Integer64Value(v))) => Some(v.to_string()),
+                        Ok(Some(FieldValue::StringValue(s))) => Some(s),
+                        Ok(Some(FieldValue::RealValue(v))) => Some(v.to_string()),
+                        Ok(Some(_)) => todo!("handle other types"),
```

Comment:
maybe we should take the chance and not crash the system when we don't support the specific type

Reply:
17e9f67e4e4b04b28fe000046fe43874cfda3764

Reply:
Is it good to return an error? This rejects the whole feature. We could make it a null value for now.